docker-compose.yml

```diff
@@ -35,6 +35,13 @@ services:
       SERVICES: dynamodb,sqs,ssm,secretsmanager,es
 
 # Utility Commands
+  composer:
+    image: composer
+    command: ["install","--ignore-platform-reqs"]
+    volumes:
+      - ~/.composer:/tmp
```

Comment:
Is this needed? I figure it's just going to involve a lot of mounted IO and affect performance

Reply:
It should allow the container to use your local composer cache (https://hub.docker.com/_/composer) so in theory save on some downloading